src/layouts/index.jsx

```diff
@@ -51,6 +51,25 @@ class Layout extends React.Component {
     }, []);
   };
 
+  getSections = () => {
+    const { pages } = this.state;
+    const sections = [];
```

Comment:
populate the pages in each section, if a sectionId is not specified they will default to section 0 which maintains backward compatibility with un-sectioned investigations